src/common/breakpoint.js

```diff
@@ -0,0 +1,12 @@
+const size = {
```

Comment:
Cool, I like the thing of having media queries with standards

Reply:
Thanks!